packages/web-app-draw-io/src/App.vue

```diff
@@ -74,6 +74,9 @@ export default defineComponent({
     this.fileExtension = this.filePath.split('.').pop()
     window.addEventListener('message', (event) => {
       if (event.data.length > 0) {
+        if (event.origin !== this.config.url) {
```

Comment:
I don't know much about draw io, but can it be hosted in a subfolder? Do we maybe need to extract the host from the url?

Reply:
The URL is currently hard-coded to https://embed.diagrams.net.

Reply:
"Returns the scheme, hostname and port of the document that caused the [onmessage](http://help.dottoro.com/ljjqtjsj.php) event."